storybooknative/src/SimpleCard/index.js

```diff
@@ -0,0 +1,52 @@
+// @flow strict
+
+import * as React from "react";
+import { View, StyleSheet } from "react-native";
+
+import styled from "../styled";
+import defaultTokens from "../defaultTokens";
+import Touchable from "../Touchable";
+
+import { Props } from "./index";
+
+const addStylesHelper = props => ({
+  borderStyle: "solid",
+  paddingLeft: `10px`,
```

Comment:
we should use paddingEnd and paddingStart instead

Reply:
Sure but with our current `styled` implementation this won't work for the web.
Because it will be converted to `padding-end` and `padding-start, and this styles property doesn't exist for the web. 

I realize that before and suggest creating some parser for react-native style to the web.
But yesterday I found that this conversion works with StyleSheet.create from the `react-native`.

So I am thinking about two solutions now:
Avoid `styled-components` and use instead of `StyleSheet.create()`.
Or try implemented this conversion on our side, maby there will be way how we can reuse `createDOMstyles` from the 'react-native-web' (https://goo.gl/U8r7or)

But first, the important question is do we really need `styled-components`?
I know it is used on the Orbit Web components, but why we need it for Orbit RNW?

Reply:
Here is an example where you can see that with `StyleSheet.create()` it's out of the box.
https://codesandbox.io/s/jn85q7k49
The `marginVertical` and `paddingEnd` are converted to web style automatically, if you use `StyleSheet` for creating styles.

Reply:
I really thought styled-components was smarter, but I guess it is not. 

I guess the reason we wanted to use styled-components was to be able to reuse some styles from web. But also adding react-native-web means we could potentially just replace existing implementation 🤔 

So the the case for re-use really is eliminated. 

I am not opposed to replacing styled-components, as I would personally not chose it for any RN project in the first place.

Reply:
BTW: What do you suggest for this MR for now?
Should I write padding for all specific platforms separately?

Reply:
`paddingEnd` for **Android** and **IOS** and `paddingRight` for **Web**

Reply:
but since it is the same padding everywhere, `padding: "10px",` would suffice?

Reply:
Otherwise, for the future, since we can have platform specific styles, we could leverage that and not care too much (for now) of duplication of styles for lack of a better solution that does not require heavy work. Thoughts?

Reply:
💯 @RobinCsl  definitely `padding: "10px"` should work. 
I feel stupid, that I didn't realize that 👽

Reply:
> I feel stupid

It's friday, so it is ok 🍺

Reply:
@FilipMessa No problem at all! That's what code reviews are for 😃 and it's Friday too 😄